packages/react-devtools-shared/src/utils.js

```diff
@@ -873,3 +879,12 @@ export function formatDataForPreview(
       }
   }
 }
+
+// Basically checking that the object only has Object in its prototype chain
+export const isPlainObject = (object: Object): boolean => {
+  const objectPrototype = Object.getPrototypeOf(object);
+  if (!objectPrototype) return true;
+
+  const objectParentPrototype = Object.getPrototypeOf(objectPrototype);
+  return objectParentPrototype == null;
```

Comment:
> Is it the only prototype with `null` as its parent prototype?

Sorry, didn't quite understand the question. I am using non-strict equals here, so essentially it will check just if `objectParentPrototype` is `undefined` or `null`

Might also just use `!objectParentPrototype` here

Reply:
Sorry for the confusion. I was just curious whether the prototype of an object is the only one that has nothing up in the chain.
This check here is good.